README.md

```diff
@@ -2,25 +2,39 @@ An R package to quantify radial bridge images
 
 Installation
 ------------
+`cranium` depends on `XQuartz`, which is not on CRAN. You can download from https://www.xquartz.org/. After install `XQuartz`, restart your R and continue `cranium` installation.
```

Comment:
This is specific to Mac OS X. This should be a subsection for operating system-specific dependencies.